kolibri/plugins/media_player/assets/src/views/index.vue

```diff
@@ -13,7 +13,7 @@
           <source :src="video.storage_url" :type="`video/${video.extension}`">
         </template>
         <template v-for="track in trackSources">
-          <track kind="captions" :src="track.storage_url" :srclang="track.lang" :label="getLangName(track.lang)" :default="isDefaultTrack(track.lang)">
+          <track kind="captions" :src="track.storage_url" :srclang="track.lang.lang_code" :label="track.lang.lang_name" :default="isDefaultTrack(track.lang.lang_code)">
```

Comment:
Note: `lang_code` is the short code for the language, so in the case of `es-mx` subtitles, this would just be `es`.

Should use `lang_id` for the long form code.